### PR TITLE
4K request fixes

### PR DESCRIPTION
**Pull request by Lucane**
Fixed some 4K functionality. Hopefully these fixes are up to par, it's my first contribution here 🙂
Getting 4K request auto-approval working was the most critical fix I think, but getting the visuals in order would be nice as well.

The following issues still need addressing:
- No separate deny button for 4K requests on the movie page (usability issue).
Workaround is to use the requests list to deny 4K requests.
- Discovery grid ignores 4K request status - doesn't show the status marker (visual issue).
- Requests list "request status" column ignores 4K status - shows every 4K-exclusive request as pending (visual issue).

Not sure where the last item gets caught on. I suspect it needs some additional logic to set the correct RequestStatus type (\Ombi.Core\Models\Requests\RequestStatus.cs), but I couldn't find where exactly.
```
<ng-container matColumnDef="requestStatus">
    <th mat-header-cell *matHeaderCellDef mat-sort-header disableClear> {{ 'Requests.RequestStatus' | translate}} </th>
    <td mat-cell id="requestedStatus{{element.id}}"  *matCellDef="let element"> {{element.requestStatus | translate}} </td>
</ng-container>
```

[2c57b30/33c721c14185d972f682fa992319bc248](https://github.com/Ombi-app/Ombi/commit/2c57b3033c721c14185d972f682fa992319bc248)
image

[572adea/05530bc7ebde48baa3300a8d9226887fc](https://github.com/Ombi-app/Ombi/commit/572adea05530bc7ebde48baa3300a8d9226887fc)
image

### CODE REVIEW

**Comment by tidusjar** [comment]
Thanks a lot for this! I'll take a look :)

File: src/Ombi/ClientApp/src/app/media-details/components/movie/panels/movie-information-panel.component.html [hunk 1]
```diff
@@ -1,136 +1,148 @@
...
-        <hr>
-        <span class="label">{{'MediaDetails.StreamingOn' | translate }}</span>
-        <div>
-        <span *ngFor="let stream of streams">
-            <img class="stream-small" [matTooltip]="stream.streamingProvider" src="https://image.tmdb.org/t/p/original{{stream.logo}}">
-        </span>
-        </div>
-    </div>
+  <div class="rating medium-font">
+    <span *ngIf="movie.voteAverage"
```
Comment (tidusjar) [inline src/Ombi/ClientApp/src/app/media-details/components/movie/panels/movie-information-panel.component.html:3]: do you mind sorting out the formatting as it's difficult to see the diff when the formatting of the file changes

Comment (Lucane) [reply src/Ombi/ClientApp/src/app/media-details/components/movie/panels/movie-information-panel.component.html]: Sorry about that! Should be better now. Visual Studio's auto-formatting can be a pain 😩

**Comment by tidusjar** [comment]
The formatting is still a bit off, The changes look ok, i'd like to pull this down manually and test before merging in

**Comment by tidusjar** [comment]
@Mergifyio update

**Comment by tidusjar** [comment]
Looking good so far! Nice work!

**Pull request merged**